src/vis/layoutUtils.ts

```diff
@@ -0,0 +1,126 @@
+import {PlotlyInfo} from './interfaces';
+
+/**
+ * Truncate long texts (e.g., to use as axes title)
+ * @param text Input text to be truncated
+ * @param maxLength Maximum text length (default: 50)
+ */
+function truncateText(text: string, maxLength = 50) {
+    return (text.length > maxLength) ? `${text.substr(0, maxLength)}...` : text;
+}
+
+export function beautifyLayout(traces: PlotlyInfo, layout: any) {
```

Comment:
Please add a function comment, because it is uclear what beautify layout does or means in this context.

Reply:
done